docs/http-authentication.rst

```diff
@@ -113,12 +48,7 @@ you will need to parse it yourself.  Everything before the first space is the
 type of authentication (probably ``Basic`` or ``Digest``), which controls which
 type of handler you'll need to construct.  The realm name is given as
 realm="foo" -- so foo would be your first argument to auth.add_password.  Other
-information in the www-authenticate header is probably safe to ignore; the
-:file:`urllib2` module will handle it for you.
```

Comment:
Don't think this part of the sentence makes sense since `urllib2` isn't being used